File: formula1-tournament-api/Models/Driver.cs
namespace formula1_tournament_api.Models
{
    public class Driver
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string RealName { get; set; }
        public int Number { get; set; }
        public Team ActualTeam { get; set; }
        public Season Season { get; set; }

        public Guid ActualTeamId { get; set; }
        public Guid SeasonId { get; set; }
        public List<Race> Races { get; set; }
        public List<Result> Results { get; set; }
    }
}

File: car-racing-tournament-web/src/app/models/driver.ts
import { Race } from "./race";
import { Result } from "./result";
import { Season } from "./season";
import { Team } from "./team";

export class Driver {
  constructor(
    public id: string,
    public name: string,
    public realName: string,
    public number: number,
    public actualTeam: Team,
    public season: Season,
    public races: Race[],
    public results: Result[]
  ) {}
}
